### ISSUE TITLE
这是内存不足了吗？

### ISSUE THREAD

**Issue by ruanqizhen** [XPixelGroup/DiffBIR#59]
我按照步骤安装，然后运行。在转换图片时出现如下错误：

RuntimeError: Unable to find a valid cuDNN algorithm to run convolution

CUDA 安装没有问题啊，这是不是表示GPU内存不够了？

**Comment by weiruihhh** [comment]
same question

**Comment by 0x3f3f3f3fun** [comment]
Yes, I have also encountered this problem when the GPU memory is not enough.

**Comment by ruanqizhen** [comment]
这个程序所需的最小 GPU 内存有多大啊？我的 8G 内存，运行不起来。

**Comment by 0x3f3f3f3fun** [comment]
至少需要11GB，对应的输入大小是512。
